crates/dslab-iaas/src/core/vm.rs

```diff
@@ -7,30 +7,35 @@ use serde::Serialize;
 use crate::core::config::SimulationConfig;
 use crate::core::load_model::LoadModel;
 
-#[derive(Clone, PartialEq, Serialize)]
+#[derive(Clone, Debug, PartialEq, Serialize)]
 pub enum VmStatus {
     Initializing,
     Running,
-    Deactivated,
+    Finished,
     Migrating,
+    FailedToAllocate,
 }
 
 impl Display for VmStatus {
     fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
         match self {
             VmStatus::Initializing => write!(f, "initializing"),
             VmStatus::Running => write!(f, "running"),
-            VmStatus::Deactivated => write!(f, "deactivated"),
+            VmStatus::Finished => write!(f, "Finished"),
```

Comment:
```suggestion
            VmStatus::Finished => write!(f, "finished"),
```
раз остальные статусы со строчной буквы